### PR TITLE
Postmark setup- email for notifying admin on user sign

### PR DESCRIPTION
**Pull request by ahweigl**
## Description/Problem

Postmark setup + notifying admin when user signs form 
**NOTE: employee 1 in seed file will need to be changed w/ iris' info for demo 

Please include a summary of the change and which issue is fixed.
Please also include relevant motivation and context.
List any dependencies that are required for this change.

## Solution

What did you change?
signFormInstance in form-instance-service

## Dependencies

[ ] This PR adds new dependencies

## Testing

Please describe how you tested this PR (manually and/or automatically)
Provide instructions so we can reproduce, and include screenshots of UI changes.

- Manual Tests?
  - Sent a ton of emails lol

### CODE REVIEW

File: apps/server/src/postmark/postmark.service.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            client.sendEmail({
+                'From': 'jfrederick@mfa.org',
+                'To': to,
+                'Subject': subject,
+                // TODO FIXME when this is uncommented the text body doesn't show up 
+                // 'HtmlBody': '<strong>Hello < /strong> dear Postmark user.',
+                'TextBody': textBody,
+                'MessageStream': 'outbound'
+            });
+            console.log('email sent');
```
Comment (kaiyangzheng) [inline apps/server/src/postmark/postmark.service.ts:28]: we can remove this

File: apps/server/src/postmark/postmark.service.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        } catch (error) {
+            console.error('Error sending email throug postmark:', error);
```
Comment (kaiyangzheng) [inline apps/server/src/postmark/postmark.service.ts:30]: this is super picky but spelling

File: apps/server/src/form-instances/form-instances.service.ts [hunk 3]
```diff
@@ -106,6 +109,7 @@ export class FormInstancesService {
         },
       },
     });
+   // this.postmarkService.sendEmail('weigl.a@northeastern.edu', 'test email: form created', 'this is a test');
```
Comment (kaiyangzheng) [inline apps/server/src/form-instances/form-instances.service.ts:112]: we can remove this

File: apps/server/src/form-instances/form-instances.service.ts [hunk 4]
```diff
@@ -379,6 +386,7 @@ export class FormInstancesService {
       data: { markedCompleted: true, markedCompletedAt: new Date() },
     });
 
+    //this.postmarkService.sendEmail('weigl.a@northeastern.edu', 'test email: form completed', 'this is a test');
```
Comment (kaiyangzheng) [inline apps/server/src/form-instances/form-instances.service.ts:389]: we can also remove this

**Review by kaiyangzheng — COMMENTED** [review]
looks good, just delete some comments and logs

File: apps/server/prisma/seed.ts [hunk 5]
```diff
@@ -155,18 +140,16 @@ async function upsertFormInstance(formInstanceData: FormInstanceData) {
...
+- 1 department (leadership team)
 - 3 signature fields ('chief', 'director', 'manager')
 - 4 positions (chief of staff, cheif financial officer, agg director, chief of learning)
 - 4 employees, one for each of the above positions
 */
-
 async function main() {
   const testFormLink =
-    'https://s29.q4cdn.com/175625835/files/doc_downloads/test.pdf';
+    'Form_Template Test _547fba69-490e-40af-8f72-70eab3815e0e';
```
Comment (AnshulShirude) [inline apps/server/prisma/seed.ts:152]: What is this number? Is it a key that should be in the .env?

**Review by AnshulShirude — APPROVED** [review]
Looks good! Just left the one question.

File: apps/server/prisma/seed.ts (hunk 5 above)
Comment (ahweigl) [reply apps/server/prisma/seed.ts:152]: @kaiyangzheng should be fine to leave for now, yeah?

**Pull request merged**